composer.json

```diff
@@ -62,6 +62,9 @@
         "mikey179/vfsstream": "^1.6.10",
         "mockery/mockery": "^1.6",
         "paragonie/csp-builder": "^2.3",
+        "phpstan/phpstan": "^1.10.30",
+        "phpstan/extension-installer": "^1.3",
+        "symplify/phpstan-rules": "^12.4",
```

Comment:
How much does this ruin your day @ADmad?

Reply:
I am not thrilled about it, but since @dereuromark has ideas on how to streamline/automate our CI tasks I don't want to hold back the changes.

I'll let you guys be the judge on whether the `simplyfy/phpstan-rules` rules bring enough value to warrant this change.

@dereuromark Do you plan to install phpstan through composer for other repos too or just this one?

Reply:
if we don't want to bloat our dev dependencies we could create a separate `composer-test.json` and set a enviromment variable to let composer know to use that one instead of the default one.
https://getcomposer.org/doc/03-cli.md#composer

Reply:
I wouldnt want to duplicate the dependencies across those files.
I think it should be fair enough for now to have those extra ones in dev, they dont leak into any dependant repo.

Reply:
So, are we good to move forward?
We still would need the github token to be added